src/app/zap-templates/zcl/data-model/chip/icd-management-cluster.xml

```diff
@@ -39,17 +39,24 @@ limitations under the License.
         <field name="AppDefinedButton" mask="0x10000"/>
     </bitmap>
 
+    <enum name="ClientTypeEnum" type="enum8">
```

Comment:
Spec does not say this is an enum8.  It says nothing about the sizing.  Needs to be fixed.

Reply:
Agreed, opened a spec issue for this last week 
https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/9281

Reply:
https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/9322